<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureBrowser</class>
 <widget class="QMainWindow" name="PictureBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图片浏览</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>647</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actionIcon_Largest"/>
    <addaction name="actionIcon_Large"/>
    <addaction name="actionIcon_Middle"/>
    <addaction name="actionIcon_Small"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionFind_Similar"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRefresh">
   <property name="text">
    <string>刷新</string>
   </property>
  </action>
  <action name="actionIcon_Largest">
   <property name="text">
    <string>超大图标</string>
   </property>
  </action>
  <action name="actionIcon_Large">
   <property name="text">
    <string>大图标</string>
   </property>
   <property name="toolTip">
    <string>Icon Large</string>
   </property>
  </action>
  <action name="actionIcon_Middle">
   <property name="text">
    <string>中图标</string>
   </property>
   <property name="toolTip">
    <string>Icon Middle</string>
   </property>
  </action>
  <action name="actionIcon_Small">
   <property name="text">
    <string>小图标</string>
   </property>
  </action>
  <action name="actionFind_Similar">
   <property name="text">
    <string>寻找相似图</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
